sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapp/AADAccessTokenGroupRolesExtractionTest.java

```diff
@@ -33,9 +33,15 @@
         new AADAuthenticationProperties.UserGroupProperties();
     private AADOAuth2UserService userService;
     private AutoCloseable autoCloseable;
+    private GroupInformation groupInformationFromGraph;
+    private Set<String> allowedGroupNames;
+    private Set<String> allowedGroupIds;
 
     @BeforeEach
     public void setup() {
+        groupInformationFromGraph = new GroupInformation();
+        allowedGroupIds = groupInformationFromGraph.getGroupsId();
```

Comment:
It should be `groupIdsFromGraph`?

Reply:
And we can call `groupInformationFromGraph.setGroupIds(...)` and `groupInformationFromGraph.setGroupNames(...)` here instead of set in each test.

Reply:
```
Mockito.lenient().when(graphClient.getGroupInformation(accessToken.getTokenValue()))
               .thenReturn(groupInformationFromGraph);
```
Can put into `setup()`, too.